testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/jwt/propagation/JWTIdentityPropagationTestCase.java

```diff
@@ -0,0 +1,425 @@
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+@ServerSetup({ JWTIdentityPropagationTestCase.PropagationSetup.class })
+public class JWTIdentityPropagationTestCase {
+
+    private static final String SINGLE_DEPLOYMENT = "single-deployment";
+    private static final String ANOTHER_SINGLE_DEPLOYMENT = "another-single-deployment";
+    private static final String NO_OUTFLOW_CONFIG = "no-outflow-config";
+    private static final String OUTFLOW_ANONYMOUS_CONFIG = "outflow-anonymous-config";
+    private static final String EAR_DEPLOYMENT_WITH_MP_JWT = "ear-deployment-mp-jwt";
+    private static final String EAR_DEPLOYMENT_WITH_EJB = "ear-deployment-ejb";
+    private static final String EAR_DEPLOYMENT_WITH_MP_JWT_SAME_DOMAIN = "ear-deployment-mp-jwt-same-domain";
+    private static final String EAR_DEPLOYMENT_WITH_EJB_SAME_DOMAIN = "ear-deployment-ejb-same-domain";
+    private static final String KEY_LOCATION = "src/test/resources/jwt/private.pem";
+
+    private static final String ROOT_PATH = "/rest/Sample/";
+    private static final String ANOTHER_ROOT_PATH = "/rest/AnotherSample/";
+    private static final String SUBSCRIPTION = "subscription";
+
+    private static final String DATE = "2017-09-15";
+    private static final String ECHOER_GROUP = "Echoer";
+    private static final String SUBSCRIBER_GROUP = "Subscriber";
+
+    private static final String PRINCIPAL_NAME = "testUser";
+    private static final String NON_EXISTING_PRINCIPAL_NAME = "nonExistingUser";
+
+    private static final String AUTHORIZATION = "Authorization";
+    private static final String BEARER = "Bearer";
+
+    private static final String EJB_SECURITY_DOMAIN = "BusinessDomain";
+    private static final String EJB_SECURITY_REALM = "BusinessRealm";
+    private static final String EJB_SECURITY_REALM_PATH = "business-realm-users";
+    private static final String EJB_SECURITY_REALM_USER = "testUser";
+    private static final String EJB_SECURITY_REALM_ADMIN_ROLE = "Admin";
+
+    private static final String ANOTHER_EJB_SECURITY_DOMAIN = "AnotherBusinessDomain";
+    private static final String ANOTHER_EJB_SECURITY_REALM = "AnotherBusinessRealm";
+    private static final String ANOTHER_EJB_SECURITY_REALM_PATH = "another-business-realm-users";
+
+    private final CloseableHttpClient httpClient = HttpClientBuilder.create().build();
+
+    @ArquillianResource
+    private URL deploymentUrl;
+
+    @Deployment(name= SINGLE_DEPLOYMENT, order = 1)
+    public static Archive<?> singleDeploymentLocal() {
+        final WebArchive war = ShrinkWrap.create(WebArchive.class, SINGLE_DEPLOYMENT + "-web.war")
+                .addClasses(JWTIdentityPropagationTestCase.class)
+                .addClasses(App.class, BeanEndPoint.class)
+                .addClasses(PropagationSetup.class)
+                .addAsManifestResource(new FileAsset(
+                                new File("src/test/resources/jwt/microprofile-config.properties")),
```

Comment:
@fjuma This needs to be:

src/test/resources/org/wildfly/test/integration/microprofile/jwt/microprofile-config.properties

Reply:
Thanks @bstansberry! I was actually trying out this change yesterday but still ran into failures so need to check if anything else has changed since I wrote the test.